dev/tools/localization/gen_l10n.dart

```diff
@@ -37,6 +37,23 @@ import 'messages_all.dart';
 /// );
 /// ```
 ///
+/// ## Update pubspec.yaml
+///
+/// Please make sure to update your pubspec.yaml to include the following
+/// packages:
+///
+/// ```
+/// dependencies:
+///   # rest of dependencies
+///
+///   flutter_localizations:
```

Comment:
Since developers will likely cut-and-paste this, maybe include a comment. For example:
```dart
///   # Internationalization support. 
///   flutter_localizations:
///     sdk: flutter
...
```

Actually, when there's a gen_l10n doc to point at, the comment should include a link.